File: remote/src/main/java/com/kelsos/mbrc/domain/Track.java
package com.kelsos.mbrc.domain;

public class Track {
  private long id;
  private String artist;
  private String title;
  private String cover;

  public Track(long id, String artist, String title, String cover) {
    this.id = id;
    this.artist = artist;
    this.title = title;
    this.cover = cover;
  }

  public long getId() {
    return id;
  }

  public String getArtist() {
    return artist;
  }

  public String getTitle() {
    return title;
  }

  public String getCover() {
    return cover;
  }
}

File: app/src/main/kotlin/com/kelsos/mbrc/domain/Track.kt
package com.kelsos.mbrc.domain

import com.kelsos.mbrc.data.db.RemoteDatabase
import com.raizlabs.android.dbflow.annotation.Column
import com.raizlabs.android.dbflow.annotation.QueryModel
import com.raizlabs.android.dbflow.structure.BaseQueryModel

@QueryModel(database = RemoteDatabase::class)
class Track() : BaseQueryModel() {
  @Column var id: Long = 0
  @Column var artist: String = ""
  @Column var title: String = ""
  @Column var cover: String = ""
}
